pkg/kubelet/cm/helpers_linux_test.go

```diff
@@ -453,6 +526,22 @@ func TestResourceConfigForPodWithCustomCPUCFSQuotaPeriod(t *testing.T) {
 			enforceCPULimits: true,
 			quotaPeriod:      tunedQuotaPeriod,
 			expected:         &ResourceConfig{CpuShares: &guaranteedShares, CpuQuota: &guaranteedTunedQuota, CpuPeriod: &tunedQuotaPeriod, Memory: &guaranteedMemory},
+			cpuManagerPolicy: "none",
+		},
+		"guaranteed-with-tuned-quota-and-cpu-manager-policy-static": {
```

Comment:
This pod will not get an exclusive (or non exclusive) CPU-set by the CPU Manager, even with policy `static`. CPU Manager will only assign exclusive an exclusive cpuset to pods where the request/limit is an interger amount of CPU.